net.cachapa.tudo.yaml

```diff
@@ -0,0 +1,24 @@
+id: net.cachapa.tudo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: tudo
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+#  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: tudo
+    buildsystem: simple
+    only-arches:
+      - x86_64
```

Comment:
also what happens with the aarch64 flatpak?

Reply:
The application only supports x86_64 at the moment.

My goal is to add aarch64 later once the x86_64 is published and running.